File: Kapil RNO 39/Java/Assign1/Employee.java
class Employee {
	
	private int id;
	private int hours;
	private int age;
	private double rate;

	public Employee(int eid, int a, int hrs, double r) {

		id = eid;
		age = a;
		hours = hrs;
		rate = r;
	}

	public void printEmployee() {
		System.out.println("----- EMPLOYEE DETAILS ----- ");
		System.out.printf(" Employee ID = %d%n",id);
		System.out.printf(" Employee Age = %d%n",age);
		System.out.printf(" Number of hours = %d%n",hours);
		System.out.printf(" Rate = %.2f%n",rate);
	}

	public double getIncome() {
		
		return hours * rate;	
	
	}

	public double getNetIncome()
	{
		return (hours * rate + (hours-8 * 100));
	}



}

File: Vishwas RNO 40/DotNet/Employee/DemoApp/Employee.cs
class Employee
{
    private int eid;
    private int age;
    private int hours;
    private double rate;

    public Employee()
    {
        eid= 100;
        age= 25;
        hours= 10;
        rate= 0.6;
    } 
    public Employee(int eid, int age, int hours, double rate)
    {
        this.eid= eid;
        this.age= age;
        this.hours= hours;
        this.rate= rate;
    }

    public double GetNetIncome(int hours, double rate) 
    {
        return hours * rate;
    }




}